ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageVideoPlayer.swift

```diff
@@ -26,11 +26,12 @@ class NewTabPageVideoPlayer {
   var player: AVPlayer?
 
   private let backgroundVideoPath: URL
-  private var didStartPlayback = false
-  private var didStartAutoplay = false
+  private var didLoadVideoAsset = false
   private var didFinishAutoplay = false
-  private var timeObserver: Any?
-  private var playerObserver: NSKeyValueObservation?
+
+  private var media25TimeObserver: Any?
+  private var media100TimeObserver: Any?
+  private var autoplayFinishedObserver: NSKeyValueObservation?
 
   private let kMaxAutoplayDurationInSeconds = 6.0
   private let kMedia25Percent = 0.25
```

Comment:
I dont think this value is not used anymore

Reply:
removed